zerver/webhooks/github/tests.py

```diff
@@ -303,6 +303,26 @@ def test_pull_request_review_requested_with_custom_topic_in_url(self) -> None:
                                           expected_message,
                                           HTTP_X_GITHUB_EVENT='pull_request')
 
+    def test_check_run(self) -> None:
+        expected_topic = u"hello-world / check runs"
+        expected_message = u"""
+Check run [randscape](http://github.com/github/hello-world/runs/4) completed (success). ([d6fde92](http://github.com/github/hello-world/commit/d6fde92930d4715a2b49857d24b940956b26d2d3))
```

Comment:
This should probably say `Check [randscape]...` -- I think verbally you don't really need the rest.